app/main.py

```diff
@@ -1 +1,28 @@
-# write your code here
+class Animal:
+    alive = []
+
+    def __init__(self, name: str, health: int = 100) -> None:
+        self.name = name
+        self.health = health
+        self.hidden = False
+        Animal.alive.append(self)
+
+    def __repr__(self) -> str:
+        return str(self.__dict__).title().replace("'", "")
+
+    @classmethod
+    def get_alive(cls) -> list:
+        return [(repr(animal) for animal in cls.alive)]
```

Comment:
The `get_alive` method currently returns a generator expression instead of a list. You should wrap the generator expression with `list()` to ensure it returns a list: `return [repr(animal) for animal in cls.alive]`.